docusaurus/docs/manage/04-cli/_login-snippet.md

```diff
@@ -0,0 +1,10 @@
+Load the current user/password into an environment variables
+
+ctrl_user=$(jq -r .username ~/.config/ziti/ziti-controller/credentials.json)

```

Comment:
Enclose BASH snippet in syntax fences like

    ```bash
    bashisms
    ```

Reply:
I'm not sure where this file came from `~/.config/ziti/ziti-controller/credentials.json`. If it's not a widely-available, standard file location then this has little value for the reader. Does this need to be updated to use the new `ziti` CLI login profiles?

Reply:
sure, it is  probably a good time to update it.